frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -204,11 +204,11 @@ export default function PatientsPage({
 
   return (
     <>
-      <NewRecordsList
+      <RecordsList
         columnDefs={ActivePatientsListColumns}
         dataName={dataName}
         defaultSort={defaultSort}
-        lazyRecordsQuery={useDashboardPatientsLazyQuery}
+        useRecordsLazyQuery={useDashboardPatientsLazyQuery}
```

Comment:
Added the prefix "use" to this props name to make it clearer that it takes in a React hook. (The "use" prefix is a common naming convention for React hooks, which are functions in React that can hold states, return variables/functions, and more.)